src/Http/Controllers/Api/Geojson/VolumeImagesController.php

```diff
@@ -0,0 +1,25 @@
+<?php
+
+namespace Biigle\Modules\Geo\Http\Controllers\Api\Geojson;
+
+use DB;
+use Biigle\Volume;
+use Biigle\Http\Controllers\Api\Controller;
+use League\Flysystem\FileNotFoundException;
+
+class VolumeImagesController extends Controller
+{
+  public function index($id){
+    $volume = Volume::findOrFail($id);
+    $this->authorize('access', $volume);
+    $images = $volume->images;
+
+    $features = collect($images)->map(function($image){
+      $feature = new \GeoJson\Feature\Feature(new \GeoJson\Geometry\Point([$image->lng, $image->lat]), Array('test'=>"test"));
```

Comment:
You should add `\GeoJson\Feature\Feature` and the like to the `use` statements at the top of the file so you can use `Feature` without the full namespace. Also, the PHP array syntax `['test' => 'test']` is cleaner than `Array('test' => 'test')`.